src/SarifWeb/Startup.cs

```diff
@@ -36,8 +37,18 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
                 app.UseHsts();
             }
+
             app.UseHttpsRedirection();
-            app.UseStaticFiles();
+
+            app.UseStaticFiles(new StaticFileOptions
+            {
+                OnPrepareResponse = ctx =>
+                {
+                    const int durationInSeconds = 60 * 60 * 24 * 30;
```

Comment:
>const int durationInSeconds = 60 * 60 * 24 * 30; [](http://example.com/codeflow?start=20&length=48)

This can be done in a cleaner and more descriptive way:
double durationInSeconds = TimeSpan.FromDays(30).TotalSeconds;

Reply:
fixed